appgate/resource_appgate_appliance.go

```diff
@@ -988,15 +984,55 @@ func readNetworkNicsFromConfig(hosts []interface{}) ([]openapi.ApplianceAllOfNet
 						ipv4networking.SetDhcp(readNetworkIpv4DhcpFromConfig(v.(map[string]interface{})))
 					}
 				}
-				// TODO do static.
+				if v := ipv4Data["static"]; len(v.([]interface{})) > 0 {
+					ipv4networking.SetStatic(readNetworkIpv4StaticFromConfig(v.([]interface{})))
+				}
```

Comment:
Note in 5.2 we are adding virtual IP for ipv4 & ipv6 here. Also MTU setting per NIC and as a DHCP option.